.github/dependabot.yaml

```diff
@@ -83,4 +109,10 @@ updates:
       - dependency-name: "@types/node"
         update-types:
           - version-update:semver-major
+      # Ignore @storybook updates, run `pnpm dlx storybook@latest upgrade` to upgrade manually
+      - dependency-name: "*storybook*" # matches @storybook/* and storybook*
+        update-types:
+          - version-update:semver-major
+          - version-update:semver-minor
+          - version-update:semver-patch
```

Comment:
Alternatively, we can add a group for `storybook` updates too.

Reply:
If dependabot had a way to run custom scripts we could. The upgrade command often includes migrations to help you move away from deprecated features to their newer counterparts, and it'd be sad to miss out on those.

Reply:
I agree. Sadly currently there isn't a way to do this. So I guess it's better to ignore storybook and do it manually. What do you think?

Reply:
agreed